src/button/template/componentStyle/responsive.js

```diff
@@ -35,6 +35,12 @@ export function buttonResponsiveStyle({ height, cardNumber = 4 } : { height? : n
                 .${ CLASS.BUTTON }.${ CLASS.BRANDING }-${ BUTTON_BRANDING.UNBRANDED } {
                     font-size: ${ max(perc(buttonHeight, 45), 10) }px;
                 }
+                
+                .${ CLASS.BUTTON } .${ CLASS.TEXT }.${ CLASS.WRAPTEXT } {
```

Comment:
Do we actually use this class anywhere in a template?

Reply:
Will remove.